service.go

```diff
@@ -205,6 +205,38 @@ func NewSeleniumService(jarPath string, port int, opts ...ServiceOption) (*Servi
 	return s, nil
 }
 
+// NewSeleniumServiceV4 starts a Selenium V4 instance in the background.
+func NewSeleniumServiceV4(jarPath string, port int, opts ...ServiceOption) (*Service, error) {
```

Comment:
I wonder if worths having a ServiceOption instead to set or not a kind of  'V4' flag. That would let use the same NewSeleniumService call